yarn.lock

```diff
@@ -5,46 +5,6 @@ __metadata:
   version: 6
```

Comment:
Hey @MeenuyD 
Should not commit yarn.lock file as you have not made any changes to the packages

Reply:
ok

Reply:
Should I create new pr

Reply:
You can fix the changes as part of the same PR. May be you can revert last commit and put another commit with the changes that are only needed as part of the PR. Also, as suggested by @ciaranha , please submit a screenshot or recording of the changes.